Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterStatisticsHelperTest.kt

```diff
@@ -20,49 +26,68 @@ class FormatterStatisticsHelperTest {
     @MockK
     private lateinit var context: Context
 
-    private val today = LocalDate()
-    private val yesterday = LocalDate().minusDays(1)
+    private val slot = slot<String>()
```

Comment:
`slot` doesn't need to be a property and can be defined as a local variable in `setUp()`

Reply:
true